File: Assignment29_5.c
#include <stdio.h>


////////////////////////////////////////////////////////////////////////////
//    Function Name :       Pattern
//    Description :         Display increasing numbers row-wise
//    Input :               Integer
//    Output :              Prints the pattern
//    Author :              Raj Samir Jadhav
//    Date :                26/11/2025
////////////////////////////////////////////////////////////////////////////


void Pattern(int iRow, int iCol)            // Function to display the pattern
{
    int i = 0, j = 0;
    int num = 0;

    for(i = 1; i <= iRow; i++)                // Loop for rows
    {
        num = i;                              // Starting number for each row

        for(j = 1; j <= iCol; j++)            // Loop for columns
        {
            printf("%d\t", num);              // Print number
            num++;                            // Increase number
        }
        printf("\n");                         // Next line
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows and columns : ");
    scanf("%d %d", &iValue1, &iValue2);

    Pattern(iValue1, iValue2);                // Function call

    return 0;
}

File: Assignment29_5.java
import java.util.Scanner;

////////////////////////////////////////////////////////////////////////////
//    Function Name :       Pattern
//    Description :         Display increasing numbers row-wise
//    Input :               Integer
//    Output :              Prints the pattern
//    Author :              Raj Samir Jadhav
//    Date :                26/11/2025
////////////////////////////////////////////////////////////////////////////

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        for(int i = 1; i <= iRow; i++)            // Loop for rows
        {
            int num = i;                           // Row starting number

            for(int j = 1; j <= iCol; j++)        // Loop for columns
            {
                System.out.print(num + "\t");
                num++;                            // Increase number
            }
            System.out.println();
        }
    }
}

class Assignment29_5
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue1, iValue2);           // Method call
    }
}
